library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java

```diff
@@ -91,10 +92,18 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
     webView.setWebChromeClient(new GeoWebChromeClient());
 
-    AirMapType mapType = AirMapType.fromBundle(getArguments());
+    // Check For MapboxWebMapType so that all Argument data can be loaded
+    Bundle arguments = getArguments();
```

Comment:
Agreed.  I was a bit unclear about this originally as `AirMapType` is Google Maps specific and there wasn't corresponding initialization in `GoogleWebViewMapFragment`.  The confusing part is [`AirMapView.getMapData()`](https://github.com/airbnb/AirMapView/blob/cc055292909e598dd4c74b6d65fad67e6f5179c9/library/src/main/java/com/airbnb/android/airmapview/AirMapType.java#L65-L70) and how it always was getting called in `WebViewFragment.onCreateView()`.